rest-service/manager_rest/test/endpoints/test_managers.py

```diff
@@ -21,27 +21,26 @@
 from cloudify_rest_client.exceptions import CloudifyClientError
 
 
-_manager1 = {
-    'hostname': 'manager1.test.domain',
-    'private_ip': '172.17.0.1',
-    'public_ip': '192.168.0.1',
-    'version': '5.0.dev1',
-    'edition': 'premium',
-    'distribution': 'centos',
-    'distro_release': 'Core',
-    'fs_sync_node_id':
-        'P56IOI7-MZJNU2Y-IQGDREY-DM2MGTI-'
-        'MGL3BXN-PQ6W5BM-TBBZ4TJ-XZWICQ2'
-}
-_manager2 = {
-    'hostname': 'manager2.test.domain',
-    'private_ip': '172.17.0.2',
-    'public_ip': '192.168.0.2',
-    'version': '5.0.dev1',
-    'edition': 'premium',
-    'distribution': 'centos',
-    'distro_release': 'Core'
-}
+_manager1 = dict(
```

Comment:
why this change? seems like just making the syntax worse for no gain :P

Reply:
Yeah the tests failed at some point with something regarding 'dictionary' issues, so I thought this might change something.
I forgot to change it back :)